mysql-test/suite/innodb/t/percona_force_encryption.test

```diff
@@ -0,0 +1,111 @@
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+
+call mtr.add_suppression("\\[Error\\] InnoDB: Can't generate new master key for tablespace encryption, please check the keyring plugin is loaded.");
+call mtr.add_suppression("\\[Error\\] InnoDB: Encryption can't find master key, please check the keyring plugin is loaded.");
+call mtr.add_suppression("\\[ERROR\\] Function 'keyring_file' already exists");
+call mtr.add_suppression("\\[ERROR\\] Couldn't load plugin named 'keyring_file' with soname 'keyring_file.*'.");
+call mtr.add_suppression("Plugin keyring_file reported");
+
+let $initial_encrypt_tables_value = `select @@innodb_encrypt_tables`;
+
+create table t1_default_explicit_on (a int) engine=innodb encryption="Y";
```

Comment:
I'm sorry if this was discussed already, but what about partitioned tables?

Reply:
They should work the same way as when you specify the encrypt flag manually - this change only adds the encrypt flag to the create table statement if it's not present, and reports an error if it is no and we want to farce encryption.

I'll add a few tests with them.

Reply:
Added test cases with partitions / tablespaces.